website/static/js/filepage/index.js

```diff
@@ -134,6 +134,16 @@ var FileViewPage = {
         });
 
         if ($osf.urlParams().branch) {
+            var fileWebViewUrl = waterbutler.buildMetadataUrl(self.file.path, self.file.provider, self.node.id, {branch : $osf.urlParams().branch});
+            $.ajax({
+                dataType: 'json',
+                async: true,
+                url: fileWebViewUrl,
+                beforeSend: $osf.setXHRAuthorization,
+                success:function(response) {
```

Comment:
As per COSDev, prefer the promise interface (`.done() or .then()`) to the `success` callback.

I will fix this when I merge.